user/serializers.py

```diff
@@ -10,15 +11,48 @@ class Meta:
         extra_kwargs = {"password": {"write_only": True, "min_length": 5}}
 
     def create(self, validated_data):
-        """Create a new user with encrypted password and return it"""
         return get_user_model().objects.create_user(**validated_data)
 
     def update(self, instance, validated_data):
-        """Update a user, set the password correctly and return it"""
         password = validated_data.pop("password", None)
         user = super().update(instance, validated_data)
         if password:
             user.set_password(password)
             user.save()
 
         return user
+
+
+class AuthTokenSerializer(serializers.Serializer):
+    email = serializers.CharField(
+        label=_("Email"),
+        write_only=True
+    )
+    password = serializers.CharField(
+        label=_("Password"),
+        style={"input_type": "password"},
+        trim_whitespace=False,
+        write_only=True
+    )
+    token = serializers.CharField(
+        label=_("Token"),
+        read_only=True
+    )
```

Comment:
if you don`t use it in validation, you can delete it

Reply:
Not agree. It doesn't depend on validation method including. 
The token field in the AuthTokenSerializer class is set by default and also set as a read_only field. It was there when we used username, but we didn't override AuthTokenSerializer but it has token field) 

However, including the token field in the serializer can be useful to provide a consistent response format to the client, and to make it clear that the authentication was successful and a token was generated.

Additionally, including the token field in the serializer class can make it easier to test the authentication process, as the generated token can be easily accessed in the test code without having to parse the response body.

So, while it is not strictly necessary to include the token field in the AuthTokenSerializer class, it is generally considered good practice to do so.